src/MapsWithSearch/HomepageMap.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import LocationSearchControl from './Input';
+import HomepageMap from '../Maps/HomepageMap';
```

Comment:
Is there a reason why the current file and the one being imported have same names? 
Do you think it is confusing ?

Reply:
hmm I could go with `HomepageMapWithSearch` for the filename, but I think that's too long and the parent directory covers that description. I'll let this be for now, but open to changing if it's confusing